apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx

```diff
@@ -219,7 +252,7 @@ export const InboundPassiveStsForm: FunctionComponent<InboundPassiveStsFormProps
                     showCertificateModal && (
                         <CertificateFormFieldModal
                             open={ showCertificateModal }
-                            certificate={ certificateDisplay }
```

Comment:
Can we check the certificateformfieldmodal UI as well with these changes? since we are passing / have passed  `null` as certificate, there might be a breaking UI.